docs/examples/waive-and-disable.md

```diff
@@ -184,9 +175,9 @@ We see this output:
       "controls": [
         "CORPORATE-POLICY_1.1"
       ],
-      "filepath": "../regula-ci-example/infra_tf/",
+      "filepath": "infra_tf",
       "platform": "terraform",
-      "provider": "aws",
+      "provider": "hashicorp/aws",
```

Comment:
I think including the `hashicorp/` prefix here is bug. The provider field should match what we were outputting for TF plans. I'm opening up a PR to fix it.

Reply:
I merged the PR that fixes this. Could you please update these when you get a chance?